scripts/lib/zulip_tools.py

```diff
@@ -359,3 +359,18 @@ def file_or_package_hash_updated(paths, hash_name, is_force, package_versions=[]
             hash_file.write(new_hash)
             return True
     return False
+
+def is_root():
+    if 'posix' in os.name and os.geteuid() == 0:
+        return True
+    return False
+
+def script_should_not_be_root(script_name = sys.argv[0]):
+    if is_root():
+        print("{} should not be run as root.  Use `su zulip` to drop root.".format(script_name))
```

Comment:
Ohh, I think the precise text of this explains some confusion we've gotten from users running e.g. `su zulip manage.py` which ends up running `manage.py` as a shell script.  I think we should expand this message a bit to address that:
```
{} should not be run as root.  Use `su zulip` to switch to the 'zulip' user before rerunning this, 
or use `su zulip -c '{} ...'` to switch users and run this as a single command.
```